### PR TITLE
Dlacheta update public dashboard

### PR DESCRIPTION
**Pull request by dariuszlacheta**

### CODE REVIEW

File: lib/builder/components/SavedQueriesSelect.js [hunk 1]
```diff
@@ -56,7 +56,9 @@ class SavedQueriesSelect extends Component {
     }
   }
 
-  selectSavedQuery = (e, index) => {
+  selectSavedQuery = (e, index) => {console.log(this.props.value)
```
Comment (apryka) [inline lib/builder/components/SavedQueriesSelect.js]: please remove log

File: lib/builder/components/ChartContainer.js [hunk 2]
```diff
@@ -277,10 +281,8 @@ class ChartContainer extends Component {
                 />
               </div>
             )}
-          </>
-        )}
         <Chart
-          key={id}
```
Comment (maciejrybaniec) [inline lib/builder/components/ChartContainer.js]: Is that change is necessary ? Using _index_ as a component _key_ could lead to strange behaviour and should be avoided.
You could read more about that [here](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318).

File: lib/builder/components/SavedQueriesSelect.js (hunk 1 above)
Comment (maciejrybaniec) [inline lib/builder/components/SavedQueriesSelect.js]: Please, remove _console.log_

File: lib/builder/components/SavedQueriesSelect.js [hunk 3]
```diff
@@ -56,7 +56,9 @@ class SavedQueriesSelect extends Component {
...
+    const { value: oldValue } = this.props;
+    this.props.changeSavedQueryList(oldValue, e);
```
Comment (maciejrybaniec) [inline lib/builder/components/SavedQueriesSelect.js]: Based on `changeSavedQueryList` action definition - the function accepts old and new value. Why in second argument the full event object is passed ?

In reducer the value is extracted from `event` - for reference `let delIndex = savedQueriesList.indexOf(el.value);`

Could we please, extract the value from `event` at container level and simplify reducer to not operate on JavaScript events ?

File: lib/reducers/rootReducer.js [hunk 4]
```diff
@@ -946,6 +948,32 @@ const editorReducer = (state = defaultData, action) => {
         }
       };
 
+    case actionTypes.CHANGE_SAVED_QUERY_LIST:
```
Comment (maciejrybaniec) [inline lib/reducers/rootReducer.js]: As we discussed - that logic could be moved easily to `thunk`.

**Pull request merged**